frontend/components/Dashboard/Settings/DetailsForm.tsx

```diff
@@ -18,21 +57,38 @@ const DetailsForm: React.FC = () => {
   const fieldError = errors[fieldErrorName]
 
   const handleDetailsSubmit = (): void => {}
-  const handleSubmitImageClick = (): void => {
-    // TODO save new image
+
+  const handleUpdateUser = (e: React.FormEvent) => {
+    e.preventDefault()
+
+    updateUser({
+      variables: {
+        userId: currentUser.id,
```

Comment:
Using cookie auth so we shouldn't need to pass this in

Reply:
Great point!